fe/fe-core/src/main/java/com/starrocks/persist/ReplicaPersistInfo.java

```diff
@@ -306,7 +312,7 @@ public void write(DataOutput out) throws IOException {
 
         out.writeInt(opType.value);
         out.writeLong(lastFailedVersion);
-        out.writeLong(0); // write a version_hash for compatibility
+        out.writeLong(minReadableVersion); // originally used as version_hash, now reused as minReadableVersion
```

Comment:
version_hash can be removed from the annotation.

Reply:
I think it's better to record the change history, I may be useful in future

Reply:
version_hash is not used from 1.9. The concept can be removed thoroughly.

Reply:
I mean when there are some compatibility issues or backport work, the code change may be confusing to new developers.